lib/wallets/wallet/impl/solana_wallet.dart

```diff
@@ -66,10 +65,8 @@ class SolanaWallet extends Bip39Wallet<Solana> {
         recipientAccount: pubKey,
         lamports: transferAmount.raw.toInt(),
       ),
-    ]).compile(
-      recentBlockhash: latestBlockhash!.value.blockhash,
-      feePayer: pubKey,
-    );
+      ComputeBudgetInstruction.setComputeUnitPrice(microLamports: 6000),
```

Comment:
Why 6000 here?

Reply:
transfer instruction is 5000 lamports and compute unit budget instruction is 1000 lamports, total of 6k lamports

Reply:
> transfer instruction is 5000 lamports and compute unit budget instruction is 1000 lamports, estimated total of 6k lamports

Include this explanation in a comment please so the seemingly random 6000 is documented in code